### ISSUE TITLE
[Tokenizer]:The “N-more” link in the tokenizer is not keyboard accessible when all tokens are overflowed

### ISSUE THREAD

**Issue by akrougliy** [UI5/webcomponents#12881]
### Bug Description

When all tokens in the tokenizer are overflowed into the “N more” link, the tokenizer does not receive keyboard focus. As a result, users cannot interact with it using the keyboard and the tokens are not accessible.

### Affected Component

Tokenizer

### Expected Behaviour

Pressing Tab and focusing on the tokenizer should expand the collapsed tokens, and place focus on the first token.

### Steps to Reproduce

1.Create tokens with long text.
2.Reduce the screen size so that all tokens overflow into the N-more link (see attachment).
3.Try to reach the tokenizer (which now consists of only N-more link) with Tab key;
...
<img width="87" height="52" alt="Image" src="https://github.com/user-attachments/assets/6598ee5d-f1d7-480f-9d1e-75dc34684ee6" />

Result:
The tokenizer does not get focus and the tokens do not expand. The user cannot access them by keyboard.

### Priority

Low

### UI5 Web Components Version

2.18.0-rc.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by StefanDimitrov04** [comment]
Hello @UI5/ui5-team-ctr-ril,

The tokenizer is not accessible via the Tab key when it is fully overflowed and only the “n more” label is visible.

Regards,
Stefan

**Comment by ndeshev** [comment]
Hello @akrougliy ,

 

Currently, according to our official UX specifications, the use case where there is no visible token (just 'n-more' link) is not supported and the width of the Tokenizer should be set to show at least one token and the 'n-more' button.

 

However, we are currently having discussions with the user experience experts to change that and to specify the case you are pointing as valid. Then we will modify the structure and behavior of the control as part of a backlog item implementation.

 

Kind regards,

Nikolay
